app/MainHie.hs

```diff
@@ -153,10 +153,10 @@ run opts = do
       Just err -> error (pdeErrorMsg err)
       Nothing -> return ()
 
-    -- let vomitOptions = GM.defaultOptions { GM.optOutput = oo { GM.ooptLogLevel = GM.GmVomit}}
-    --     oo = GM.optOutput GM.defaultOptions
-    -- let ghcModOptions = vomitOptions { GM.optGhcUserOptions = ["-Wall"]  }
-    let ghcModOptions = GM.defaultOptions { GM.optGhcUserOptions = ["-Wall"]  }
+    let vomitOptions = GM.defaultOptions { GM.optOutput = oo { GM.ooptLogLevel = GM.GmVomit}}
```

Comment:
Probably best to disable this before merging into master.

And best of all to make it subject to a flag